datetimepicker/src/main/java/com/taransit/datetimepicker/dialog/DateTimePickerDialog.kt

```diff
@@ -145,18 +129,22 @@ class DateTimePickerDialog(
                 calendar.set(Calendar.MINUTE, timePicker.minute)
             }
 
-            listener?.onDateTimeSet(calendar, Direction.from(viewPager.currentItem))
+            mListener?.onDateTimeSet(calendar, pages.getOrNull(viewPager.currentItem))
             dismissAllowingStateLoss()
         }
+
+        cancel.setText(negativeButtonText)
+        confirm.setText(positiveButtonText)
     }
 
     override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
-        val dialog = super.onCreateDialog(savedInstanceState)
-        dialog.requestWindowFeature(Window.FEATURE_NO_TITLE)
-        val background = dialog.window?.decorView?.background as? InsetDrawable
-        background?.alpha = 0
-        dialog.window?.setBackgroundDrawable(background)
-        return dialog
+        return super.onCreateDialog(savedInstanceState).apply {
+            requestWindowFeature(Window.FEATURE_NO_TITLE)
+            val background = window?.decorView?.background as? InsetDrawable
+            background?.alpha = 0
+            window?.setBackgroundDrawable(background)
+            window?.setLayout(resources.getDimensionPixelSize(R.dimen.dialog_width), -2)
```

Comment:
-2 natvrdo v kóde, netuším, čo to znamená, dať do premennej?

Reply:
Fixed